src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
```

Comment:
- [ ] It would be better to populate the books array with data, for example;
```
books: [
    {
      item_id: "item1",
      title: "The Hunger Games",
      author: "Suzanne Collins",
      category: "Fiction",
    },
    {
      item_id: "item2",
      title: "Dune",
      author: "Frank Herbert",
      category: "Fiction",
    },
    {
      item_id: "item3",
      title: "Capital in the Twenty-First Century",
      author: "Suzanne Collins",
      category: "Nonfiction",
    },
  ],
```
Instead of passing the object property values to the book component and leaving the books array empty

Reply:
Thank you for this review. Very helpful! 😃